### DISCUSSION TITLE
Initial Installation headache: gifsicle

### DISCUSSION THREAD

**Discussion by ea42gh** [ahuang11/ahlive#18] [General]
Got myself in trouble right away: tried to run a first example on Windows;
I wanted to reduce the noise:
had to do some web searches for gifsicle.

Now that that is running, I wanted to get rid of the remaining warnings and completion indicator.
How might I go about that?

**Comment by ahuang11** [comment]
Good points! I'll create issues for these.

**Reply by ahuang11** [reply]
This should be fixed now.
